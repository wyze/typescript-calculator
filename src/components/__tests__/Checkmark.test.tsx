import { h } from 'preact'
import Checkmark from '../Checkmark'
import snapshot from './utils/snapshot'

describe('<Checkmark />', () => {
  it('renders', () => snapshot(<Checkmark />))
})
