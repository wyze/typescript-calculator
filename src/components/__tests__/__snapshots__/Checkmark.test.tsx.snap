// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Checkmark /> renders 1`] = `
<Svg>
  <circle
    cx="32"
    cy="32"
    fill="#4bd37b"
    r="30"
  />
  <path
    d="m46 14l-21 21.6-7-7.2-7 7.2 14 14.4 28-28.8z"
    fill="#fff"
  />
</Svg>
`;
